### PR TITLE
Srcset Bug Fix

### PR DESCRIPTION
**Pull request by annebyrne**
A combination of typos ( 😨  come on now Anne!), and poor refactoring meant that a bug was introduced where the srcset images weren't correctly populated and so that images would not render.

The secondary bug was in missing the write() import, which meant that whilst the templates worked in preview testing they haven't been rendering as a creative on site with a test line item.

cc @regiskuckaertz @Ap0c

### CODE REVIEW

File: src/_shared/js/images.js [hunk 1]
```diff
@@ -1,33 +1,34 @@
...
 export function insertBetweenComments(sources) {
 
 	let pictures = Array.from(document.getElementsByTagName('picture'));
 
   return write(() => {
-		pictures.forEach((picture, index) => picture.insertBefore(sources[index], picture.firstChild));
+		pictures.forEach((picture, index) => {
+      picture.insertBefore(sources[index][0], picture.firstChild);
+      picture.insertBefore(sources[index][1], picture.firstChild);
+    });
```
Comment (Ap0c) [inline src/_shared/js/images.js]: I don't think this works... There's only one `picture` tag with six sources, not three `picture` tags with two sources each. As a result this only inserts the first two `source` elements into the page and skips out the last four.

File: src/capi-single-supported/index.js [hunk 2]
```diff
@@ -62,7 +62,7 @@ function populateCard(responseJson) {
           </picture>
```
Comment (Ap0c) [inline src/capi-single-supported/index.js]: You need to have the IE9 comments already [present](https://github.com/guardian/commercial-templates/blob/capi-multiple/src/capi-multiple-paidfor/index.html#L44) in the page in order be inserting between them with the `insertBetweenComments` function.

**Comment by annebyrne** [comment]
@regiskuckaertz @Ap0c 🚢   ?

File: src/_shared/js/images.js [hunk 3]
```diff
@@ -18,18 +18,25 @@ export function addSourceset(responseJson) {
...
-	let pictures = Array.from(document.getElementsByTagName('picture'));
+    let picture = document.getElementsByTagName('picture')[0];
 
-  return write(() => {
-		pictures.forEach((picture, index) => {
-      picture.insertBefore(sources[index][0], picture.firstChild);
-      picture.insertBefore(sources[index][1], picture.firstChild);
+    return write(() => {
+      sources.forEach((source, index) => {
+        picture.insertBefore(source[0], picture.firstChild);
```
Comment (regiskuckaertz) [inline src/_shared/js/images.js]: Let's see what this gives us, but we might want to use `appendChild` instead of `insertBefore` here.

File: src/_shared/js/images.js [hunk 4]
```diff
@@ -18,18 +18,25 @@ export function addSourceset(responseJson) {
...
+      });
     });
-	});
+  }
+
+export function buildImages(sources) {
+
+  let imageTag = document.getElementsByTagName('img')[0];
+
+  (imageTag.srcset === '')? insertSrcset(sources) : imageTag;
```
Comment (regiskuckaertz) [inline src/_shared/js/images.js]: Better: `'srcset' in imageTag ? ...`

File: src/_shared/js/messages.js [hunk 5]
```diff
@@ -19,7 +19,7 @@ export function getIframeId() {
             } catch(_) { return; }
 
             const keys = Object.keys(json);
-            if( keys.length !== 3 || !keys.includes('id') || !keys.includes('host') ) return;
```
Comment (regiskuckaertz) [inline src/_shared/js/messages.js]: Feels like this does not belong to this PR

File: src/_shared/scss/_adverts.scss [hunk 6]
```diff
@@ -324,13 +324,8 @@
 
 .adverts--legacy-inline {
     padding: 0;
-    width: $gs-gutter * 6.5;
```
Comment (regiskuckaertz) [inline src/_shared/scss/_adverts.scss]: Ibid.

File: src/manual-inline/index.html [hunk 7]
```diff
@@ -0,0 +1,19 @@
+<aside class="adverts adverts--manual adverts--legacy-inline adverts--[%Tone%] adverts--tone-[%Tone%]" data-link-name="creative | ad inline manual | [%OmnitureId%]">
```
Comment (regiskuckaertz) [inline src/manual-inline/index.html]: Ibid.

File: src/manual-inline/index.js [hunk 8]
```diff
@@ -0,0 +1,5 @@
+import { getIframeId, resizeIframeHeight,getWebfonts } from '../_shared/js/messages.js';
```
Comment (regiskuckaertz) [inline src/manual-inline/index.js]: Ibid.

File: src/_shared/js/images.js (hunk 3 above)
Comment (Ap0c) [inline src/_shared/js/images.js]: We're going to refactor a lot of this together with the multiple in a separate PR. Currently just getting our versions merged so we're not working in parallel.
